### DISCUSSION TITLE
RGB NFT vs other NFTs

### DISCUSSION THREAD

**Discussion by UkolovaOlga** [LNP-BP/LNPBPs#101] [Q&A]
## RGB NFT vs other NFTs:
- no need in costly blockchain storage
- no need in IPFS (which is also unencrypted)
- no need in special data management solution
- no need in trusting websites to keep the data
- embedded DRM-type encryption and ownership management
- infrastructure for backups utilizing Lightning Network
- ways to monetize content (sell not the NFT itself, but access to the content, multiple times)

### Question
Media files publicly available but only as encrypted blobs with (maybe) low-res "fully public" versions. The nft "sale" happens when the buyer pays for the decryption key associated with a particular media file.
If I'm right about that, then what's stopping the nft recipient from using the decryption key to decrypt the media file and then republish it for free or resell it?

### Answer
Nothing can stop him. However he would not be able to do that for payment, since he would not be able to prove to the buyer his ownership of the content, meaning that in trustless environment most buyers will prefer buying content from those who can prove that they have it trustlessly (with RGB ownership).

In general, the way DRM system works today is very far from individual/author/owner-centric, leading to appearance of huge industry of distributors and labels _and_ impossibility to _buy into ownership_ of some certain content. So the assumption is, that removing the ability to _prevent_ copyright technically (current DRM) and use instead economical incentives is much more sustainable and will lead to the healthier ecosystem. That is what can be expected from the RGB NTF design.

**So, the change with RGB (in legal terms) is:**

- when content creator sells his content, he (1) does that directly to the buyer and (2) he sells the right to copy (i.e. "copyright”) with it (which is the actual ownership over a piece of content) - and he can do that multiple times. RGB provides distribution channel (LN+Bifrost), decentralized storage (Bifrost+Storm), payment (BTC, lBTC or RGB stablecoins like USDT) and protection of the content from being accessed by non-buyers (client-side-validation with encryption and confidentiality) for that.

- when content owner sells content, he has two options: (1) sell access to the content, for smaller money than when the content is sold, so he sells “usage rights”, and the new content user will be able to use the content/copy multiple times, but will have a challenge of selling it (still being able to distribute it for free) and (2) pass the ownership rights further, the same way the content creator did.

This indeed does not provide _copy_ protection at technical level. But what it does, it provides _free public access_ protection at technical level (until one of the content users will publish it for free).

That is weaker publisher guarantees comparing to DRM, but stronger comparing to any other NFT or alternative out there (and the same strong guarantees as with software licenses back in old days).

At the end of the day, arguably, there is a probability that if we take game theory & economy into account (and not only technical means) that will still give _more_ profit to the content producers because we remove intermediaries (lables/distributors) taking 90% of profitability from the equation + provide economic incentives to pay to owners, and not cheating content users. But it will be interesting to see what @sabina_sa thinks of that (she is economist/game theory expert).

_From discussion on Telegram between @supertestnet, @dr-orlovsky._

**Comment by iafi01** [comment]
Really interesting!
